torch/_inductor/mkldnn_lowerings.py

```diff
@@ -521,6 +527,8 @@ def qlinear_unary(
                 x_scale.realize()
             if not isinstance(x_zp, ir.TensorBox):
                 assert type(x_zp) == int
+                if x.get_dtype() == torch.int8 and x_zp == 0:
```

Comment:
We don't support the case: act is int8 and zp != 0. How do we make sure it doesn't happen?

Reply:
We support it